app/Repositories/Eloquents/BookEloquentRepository.php

```diff
@@ -20,6 +20,7 @@ class BookEloquentRepository extends AbstractEloquentRepository implements BookR
         'description',
         'avg_star',
         'slug',
+        'author'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookEloquentRepository.php</br>**Line**: 23</br>Comma required after last value in array declaration